pkgs/development/python-modules/pvlib/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildPythonPackage, fetchPypi, numpy, pandas, pytz, six, pytest }:
+
+buildPythonPackage rec {
+  pname = "pvlib";
+  version = "0.5.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1897v9qq97nk5n0hfm9089yz8pffd42795mnhcyq48g9bsyap1xi";
+  };
+
+  checkInputs = [ pytest ];
+  propagatedBuildInputs = [ numpy pandas pytz six ];
+
+  # Currently, the PyPI tarball doesn't contain the tests. When that has been
+  # fixed, enable testing. See: https://github.com/pvlib/pvlib-python/issues/473
+  checkPhase = "true";
```

Comment:
if you want to disable tests, use `doCheck = false;`

Reply:
Done.